### PR TITLE
Feat: automatic generation of bus actors file

### PR DESCRIPTION
**Pull request by jitsedesmet**

### CODE REVIEW

**Comment by jitsedesmet** [comment]
Alright, so the current version works but, it uses the description of the actors and busses as provided by the package.json file. This description is... rather bad for most packages.

There seems to be a line in almost every README.md file that starts looks like:
"A {bus-name} actor {useful-info}."

I could try and look for that line and put in the website too, but idk, maybe the package descriptions should just be better?
What do you think?

**Comment by rubensworks** [comment]
The "first" line in the README of each package could work indeed.
But for packages that don't have a good README yet, perhaps we should change the README so it matches the one currently shown on the website.

Also, I guess we can just autogenerate the "Subscribed actors need to implement ..." part.

**Comment by jitsedesmet** [comment]
Screenshot of zoomed out page 
image

File: pages/docs/modify/advanced/buses.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    </div>
+    </Template>
+  )
+}
+
+async function fetchFile(url) {
+    return await (await fetch(url)).text();
+}
+
+async function fetchJson(url) {
```
Comment (rubensworks) [inline pages/docs/modify/advanced/buses.js:129]: The Fetch API has a JSON method :-) https://developer.mozilla.org/en-US/docs/Web/API/Response/json

File: pages/docs/modify/advanced/buses.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            {actorsInfo}
+            </tbody>
+        </table>
+    </>
+}
+
+export default function Buses({ bussesInfo }) {
+  const busInfo = bussesInfo.map(busInfo => functionBusInfo(busInfo));
+  return (
+    <Template key={'/docs/modify/advanced/busbis/'}>
```
Comment (rubensworks) [inline pages/docs/modify/advanced/buses.js:80]:
```suggestion
    <Template key={'/docs/modify/advanced/buses/'}>
```

File: pages/[...slug].js [hunk 3]
```diff
@@ -5,10 +5,20 @@ import BlogIndex from "../components/BlogIndex";
 import BreadCrumbs from "../components/Breadcrumbs";
 import Markdown from "../components/Markdown";
 import React from 'react';
+import Template from "./template";
+
+const additionalMattersData = [
+    {
+        path: '/docs/modify/advanced/busbis/',
```
Comment (rubensworks) [inline pages/[...slug].js:12]:
```suggestion
        path: '/docs/modify/advanced/buses/',
```

File: pages/[...slug].js [hunk 4]
```diff
@@ -5,10 +5,20 @@ import BlogIndex from "../components/BlogIndex";
...
+        sortKey: '/docs/2_modify/advanced/busbis/',
```
Comment (rubensworks) [inline pages/[...slug].js:13]:
```suggestion
        sortKey: '/docs/2_modify/advanced/buses/',
```

**Comment by rubensworks** [comment]
Nice, great work @jitsedesmet!

**Pull request merged**